training/models/event_event.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+from openerp import api, fields, models
+from .common import M
+from .. import exceptions
+
+
+class Event(models.Model):
+    """Expand events with training actions.
+
+    Events with a training type and a training action are considered training
+    groups.
+    """
+    _inherit = "event.event"
+
+    material_ids = fields.Many2many(
+        M % "material",
+        string="Materials",
+        states={"done": [("readonly", True)]},
+        help="These should be delivered to every student in this training.")
+    training_action_id = fields.Many2one(
+        M % "action",
+        "Training action",
+        states={"done": [("readonly", True)]},
+        help="Training action of this event, if it is a training group.")
+    training_mode = fields.Boolean(
+        compute="_compute_training_mode",
```

Comment:
It's not that simple. Users can use events from Marketing menu (as always) or from Training menu. Marketing allows you to convert an event that was a training group to a normal event, but the Training menu does not, because thanks to this trick the course field is required when using that menu.

For what you say, the logic is similar, but depends on *where was the record created*, not *where is the user right now*, which is much easier to understand, user POV.